libinteract/libinteract.py

```diff
@@ -357,30 +357,30 @@ def do_potential(kbp_atomlist,
             calc_potential_func(distances = distances,
                                 ordered_sparses = ordered_sparses,
                                 kbT = kbT)
+
+        # Update the frame number
+        numframe += 1
     
     # Divide the scores for the lenght of the trajectory
     scores /= float(numframes)
-    
-    # Create the output string
-    outstr = ""
-    
-    # Set the format for the representation of each pair of
-    # residues in the output string
-    outstr_fmt = "{:s}-{:s}{:d}:{:s}-{:s}{:d}\t{:.3f}\n"
+
+    # Create an empty list to store the final table
+    table = []
     
     for i, score in enumerate(scores):
         if abs(score) > 0.000001:
             
-            # Update the output string
-            outstr +=  \
-                outstr_fmt.format(\
-                    pdb.residues[residue_pairs[i][0].ix].segment.segid,
-                    pdb.residues[residue_pairs[i][0].ix].resname,
-                    pdb.residues[residue_pairs[i][0].ix].resid,
-                    pdb.residues[residue_pairs[i][1].ix].segment.segid,
-                    pdb.residues[residue_pairs[i][1].ix].resname,
-                    pdb.residues[residue_pairs[i][1].ix].resid,
-                    score)
+            # Update the output table
+            table.append(\
+                 (pdb.residues[residue_pairs[i][0].ix].segment.segid,
+                  pdb.residues[residue_pairs[i][0].ix].resnum,
+                  pdb.residues[residue_pairs[i][0].ix].resname,
+                  ".".join(kbp_atomlist),
```

Comment:
we can replace this just with `sidechain` similar to how it is for HC now - otherwise the resulting dat is really ugly

Reply:
Sure thing!